classes/dataflow.php

```diff
@@ -664,4 +665,26 @@ public function set_var($name, $value) {
         // Updates the stored config.
         $this->config = Yaml::dump((array) $config, 2, 4, Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK);
     }
+
+    /**
+     * Saves the current config to the tool_dataflows_versions table
+     *
+     */
+    public function save_config_version() {
+        if ($this->confighash === null || $this->confighash === "") {
+            global $DB;
+
+            $config = $this->export();
+            $configyaml = Yaml::dump((array)$config, 2, 4, Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK);
```

Comment:
Can you please ensure there is a space before and after the `(cast)` as shown in the example referenced below:

```suggestion
            $configyaml = Yaml::dump((array) $config, 2, 4, Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK);
```

https://www.php-fig.org/psr/psr-12/#61-unary-operators